### PR TITLE
Added pays.host presence

### PR DESCRIPTION
**Pull request by hiitsdan**
image
image
image

### CODE REVIEW

File: websites/P/pays.host/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	"tags": [
+		"sharex",
+		"pays",
+		"host",
+		"image",
+		"host"
+	],
+	"category": "other",
+	"iframe": false,
+	"readLogs": false
```
Comment (Bas950) [inline websites/P/pays.host/dist/metadata.json:29]: You dont need to set these, they are false by default.

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/P/pays.host/presence.ts [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+    }
+
+    const welcomeMessageElement = document.getElementsByClassName("welcomeText")[0];
+
+    if (welcomeMessageElement) {
+      const welcomeMessage = welcomeMessageElement.textContent,
+        username = welcomeMessage.replace("Welcome, ", "");
+      presenceData.smallImageKey = "user";
+      presenceData.smallImageText = `Logged in as ${username}`;
+    }
```
Comment (Bas950) [inline websites/P/pays.host/presence.ts:102]: make a privacy setting for this or hide it.

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/P/pays.host/presence.ts [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+      }
+
+      case "/configs": {
+        presenceData.state = "Viewing configs page";
+        break;
+      }
+
+      case "/domains": {
+        const domainCount = document.getElementById("domain-count").innerText;
+        presenceData.state = "";
```
Comment (ririxidev) [inline websites/P/pays.host/presence.ts:46]: don't set "", do ``delete presenceData.state``

File: websites/P/pays.host/presence.ts [hunk 4]
```diff
@@ -0,0 +1,285 @@
...
+
+      case "/home": {
+        presenceData.state = "Viewing home page";
+        break;
+      }
+
+      case "/domains": {
+        const domainCountString = document.querySelector("main div p:nth-child(3)").textContent,
+          [domainCount] = /\d+/.exec(domainCountString);
+        presenceData.state = "";
```
Comment (ririxidev) [inline websites/P/pays.host/presence.ts:122]: ``delete presenceData.state``

**Review by ririxidev — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]

**Review by ririxidev — APPROVED** [review]

**Pull request merged**